DE Translation (#694)
Heegu-sama/Homm3BG: sections/combat.tex

\vspace*{-3em}

\subsection*{\pagetarget{Quick}{Quick Combat}}\index{Quick Combat}

> Falls dein Helden-Level höher than a Field's Difficulty when Combat against Neutral Units would begin, \textbf{no Combat} takes place.

Reviewer: ```suggestion
If your Hero's Level is higher than a Field's Difficulty when Combat against Neutral Units would begin, \textbf{no Combat} takes place.
```
